content/en/blog/_posts/2023-07-24-unknown-version-interoperability-proxy-alpha.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Unknown Version Interoperability Proxy (Alpha)"
+date: 2023-05-24
+slug: kubernetes-1-28-feature-unknown-version-interoperability-proxy-alpha
+---
+
+**Author:** Richa Banker (Google)
+
+This blog describes “Unknown Version Interoperability Proxy”, a new alpha feature in Kubernetes 1.28. Unknown Version Interoperability Proxy enables a resource request to be served by the correct apiserver in cases where there are multiple apiservers at varied version in a cluster, ex: during an upgrade/downgrade of a cluster, when a runtime config changes and a rollout occurs.
+
+## What problem does this solve?
+When a cluster undergoes an upgrade or downgrade, the kube-apiservers existing at different versions in that scenario can serve different sets (groups, versions, resources) of built-in resources. A resource request made in this scenario may be served by any of the available apiservers, potentially resulting in the request ending up at an apiserver that may not be aware of the requested resource; consequently it being served a 404 not found error which is incorrect.
+
+Furthermore, incorrect serving of the 404 errors can lead to serious consequences such as namespace deletion being blocked incorrectly or objects being garbage collected mistakenly.
```

Comment:
nit: I'd merge this sentence with the previous paragraph. It's a logical extension of the previous thought.

Reply:
Done